tests/phpunit-admin.xml

```diff
@@ -0,0 +1,15 @@
+<phpunit bootstrap="bootstrap-admin.php">
+    <php>
+      <server name="kernel.environment" value="test" />
```

Comment:
It was setup in the xml config before I realize I needed it in the bootstrap as well. I'll remove it from the XML file.